pkgs/development/python-modules/sudospawner/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, jupyterhub
+, notebook
+, escapism
+}:
+
+buildPythonPackage rec {
+  pname = "sudospawner";
+  version = "0.5.2";
```

Comment:
nixos modules usually take a while for them to get merged. I would recommend factoring this out to another PR.

Reply:
`sudospawner` might be useless to merge by itself without necessary support in the module. However, `openapi-core` and `jupyterlab_server` changes are indeed useful on their own, so let's split them.

Reply:
https://github.com/NixOS/nixpkgs/pull/154617 and https://github.com/NixOS/nixpkgs/pull/154618.